File: src/main/java/org/mcausc78/hucooldown/HuCooldown.java
package org.mcausc78.hucooldown;

import org.bukkit.plugin.java.JavaPlugin;

import java.util.HashMap;

public final class HuCooldown extends JavaPlugin {
    private static HuCooldown instance;
    private static HashMap<String, Long> storage;
    @Override
    public void onEnable() {
        instance = this;
        saveDefaultConfig();
        getServer().getPluginManager().registerEvents(new CmdListener(), this);
    }

    @Override
    public void onDisable() {}

    public static HuCooldown getInstance() {
        return instance;
    }

    public static HashMap<String, Long> getStorage() {
        return storage;
    }
}

File: src/main/kotlin/org/mcausc78/hucooldown/HuCooldown.kt
package org.mcausc78.hucooldown

import org.bukkit.plugin.java.JavaPlugin

class HuCooldown : JavaPlugin() {
    override fun onEnable() {
        instance = this
        saveDefaultConfig()
        server.pluginManager.registerEvents(HuListener(), this)
    }

    override fun onDisable() {}

    companion object {
        var instance: HuCooldown? = null
            private set
        val storage: HashMap<String, Long>? = null
    }
}